packages/fe-fpm-writer/src/page/custom.ts

```diff
@@ -91,6 +91,14 @@ export function generate(basePath: string, data: CustomPage, fs?: Editor): Edito
     if (!fs.exists(viewPath)) {
         fs.copyTpl(join(root, 'ext/View.xml'), viewPath, config);
     }
+    // i18n.properties
+    const i18nPath = join(basePath, 'webapp', 'i18n', 'i18n.properties');
+    const i18TemplatePath = join(root, 'i18n', 'i18n.properties');
+    if (fs.exists(i18nPath)) {
```

Comment:
@devinea 

One concern about hardcoded path to `i18n` file,
there can be scenario where we have `manifest.json` like:
```
"sap.ui5": {
        "models": {
            "i18n": {
                "type": "sap.ui.model.resource.ResourceModel",
                "uri": "custom-i18n2/i18n.properties"
            },
```

Then if there is such manifest and I create custom page using fpm-writer - in result new `i18n` entry or new file is created within `i18n/i18n.properties`. Maybe we could try detect path based on `manifest.json`?
I think custom i18n paths are rare scenario, not fully sure how critical it is

Reply:
@815are Thanks. Addressed in https://github.com/SAP/open-ux-tools/pull/1475/commits/0991f7d897b654b19de30916f8c99395b7347cfa